backend/test/game.e2e-spec.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpStatus, INestApplication, ValidationPipe } from '@nestjs/common';
```

Comment:
general question is there a command that generates the e2e tests or do you create the file manually?